packages/server/events-processing-platform/subscriptions/notifications/notifications_subscriber.go

```diff
@@ -0,0 +1,246 @@
+		contactevent.ContactOrganizationLinkV1:
+		return nil
+
+	case orgevents.OrganizationCreateV1:
+		return nil
+	case orgevents.OrganizationUpdateV1: // TODO: implement on update send email
+		return nil
+	case orgevents.OrganizationPhoneNumberLinkV1,
+		orgevents.OrganizationEmailLinkV1,
+		orgevents.OrganizationLocationLinkV1,
+		orgevents.OrganizationLinkDomainV1,
+		orgevents.OrganizationAddSocialV1,
+		orgevents.OrganizationHideV1,
+		orgevents.OrganizationShowV1,
+		orgevents.OrganizationRefreshLastTouchpointV1,
+		orgevents.OrganizationRefreshArrV1,
+		orgevents.OrganizationRefreshRenewalSummaryV1,
+		orgevents.OrganizationUpsertCustomFieldV1,
+		orgevents.OrganizationAddParentV1,
+		orgevents.OrganizationRemoveParentV1,
+		orgevents.OrganizationUpdateOnboardingStatusV1,
+		orgevents.OrganizationRequestRenewalForecastV1,
+		orgevents.OrganizationRequestNextCycleDateV1,
+		orgevents.OrganizationUpdateRenewalLikelihoodV1,
+		orgevents.OrganizationUpdateRenewalForecastV1,
+		orgevents.OrganizationUpdateBillingDetailsV1,
+		orgevents.OrganizationRequestScrapeByWebsiteV1:
+		return nil
+
+	case userevents.UserCreateV1,
+		userevents.UserUpdateV1,
+		userevents.UserPhoneNumberLinkV1,
+		userevents.UserEmailLinkV1,
+		userevents.UserJobRoleLinkV1,
+		userevents.UserAddPlayerV1,
+		userevents.UserAddRoleV1,
+		userevents.UserRemoveRoleV1:
+		return nil
+
+	case locationevents.LocationCreateV1,
+		locationevents.LocationUpdateV1,
+		locationevents.LocationValidationFailedV1,
+		locationevents.LocationValidationSkippedV1,
+		locationevents.LocationValidatedV1,
+		jobroleevents.JobRoleCreateV1:
+		return nil
+
+	case ieevent.InteractionEventRequestSummaryV1,
+		ieevent.InteractionEventRequestActionItemsV1,
+		ieevent.InteractionEventReplaceSummaryV1,
+		ieevent.InteractionEventReplaceActionItemsV1,
+		ieevent.InteractionEventCreateV1,
+		ieevent.InteractionEventUpdateV1:
+		return nil
+
+	case logentryevents.LogEntryCreateV1,
+		logentryevents.LogEntryUpdateV1,
+		logentryevents.LogEntryAddTagV1,
+		logentryevents.LogEntryRemoveTagV1:
+		return nil
+
+	case commentevent.CommentCreateV1,
+		commentevent.CommentUpdateV1:
+		return nil
+
+	case issueevent.IssueCreateV1,
+		issueevent.IssueUpdateV1,
+		issueevent.IssueAddUserAssigneeV1,
+		issueevent.IssueRemoveUserAssigneeV1,
+		issueevent.IssueAddUserFollowerV1,
+		issueevent.IssueRemoveUserFollowerV1:
+		return nil
+
+	case opportunityevent.OpportunityCreateV1,
+		opportunityevent.OpportunityUpdateNextCycleDateV1,
+		opportunityevent.OpportunityUpdateV1,
+		opportunityevent.OpportunityCreateRenewalV1,
+		opportunityevent.OpportunityUpdateRenewalV1,
+		opportunityevent.OpportunityCloseWinV1,
+		opportunityevent.OpportunityCloseLooseV1:
+		return nil
+
+	case contractevent.ContractCreateV1,
+		contractevent.ContractUpdateV1,
+		contractevent.ContractRolloutRenewalOpportunityV1,
+		contractevent.ContractUpdateStatusV1:
+		return nil
+
+	case servicelineitemevent.ServiceLineItemCreateV1,
```

Comment:
i think we can specify only our use case on a case and use the default to return nil, kind of what's below. no need to treat all the events just to return nil

case contractevent.OwnerChangedWhatever: do the logic; break
default: return nil

Reply:
no problem, I don't actually know which event we want to handle. It's when a user is added as owner to an organisation. Do you know what event it is?